fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkexport/patient/ChunkReader.java

```diff
@@ -269,7 +300,11 @@ public Object readItem() throws Exception {
 
             List<String> patientIds = resources.stream().filter(item -> item.getId() != null).map(item -> item.getId()).collect(Collectors.toList());
             if (patientIds != null && patientIds.size() > 0) {
-                fillChunkDataBuffer(patientIds);
+                if (fhirResourceType.equalsIgnoreCase("patient")) {
+                    fillChunkPatientDataBuffer(resources);
+                } else {
+                    fillChunkDataBuffer(patientIds);
+                }
```

Comment:
This may be candidate to go in a common class as it's consistent with some of the code in group

Reply:
No need to refactor here, just a thought going forward

Reply:
also done